leku/build.gradle

```diff
@@ -88,13 +88,13 @@ dependencies {
 	  exclude group: "com.android.support"
   }
 
-  implementation ("com.google.android.libraries.places:places-compat:2.2.0") {
+  implementation ("com.google.android.libraries.places:places:2.2.0") {
     exclude group: "com.android.support"
   }
 
   implementation "com.google.maps:google-maps-services:0.2.9"
 
-  implementation 'pl.charmas.android:android-reactive-location2:2.1@aar'
+  implementation 'com.github.MarsXan:Android-ReactiveLocation:2.1.0'
```

Comment:
We have to change the old library to this modified modified one because the old library does not supported the new Google Places API. This is kind of a workaround until we decide what to do with this code.